Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
How about adding:

```
RUN sed -i '/use_memcached = / s/False/True/' mote/config.py
RUN sed -i sed -i 's|json_cache_location = "/var/cache/httpd/mote/cache.json"|json_cache_location = "./cache.json"|' config.py
```

to make sure the user / developer don't have to make any modifications to the project while using the containerized version with the file cache.

Reply:
Yep, sounds good. Would make the dev experience a bit more convenient.